src/Resources/views/Form/form_admin_fields.html.twig

```diff
@@ -357,7 +357,9 @@ file that was distributed with this source code.
 
 {% block form_row %}
     {% set show_label = show_label ?? true %}
-    <div class="form-group{% if errors|length > 0 %} has-error{% endif %}" id="sonata-ba-field-container-{{ id }}">
+    {% set row_id = 'sonata-ba-field-container-'~id %}
+    {% set row_class = (row_attr.class|default('') ~ ' form-group' ~ ((not compound or force_error|default(false)) and not valid ? ' has-error'))|trim %}
```

Comment:
You changed the check for the `has-error`.
Previously it was `{% if errors|length > 0 %}`.

Can you explain why ?

Reply:
I copied it from the Symfony implementation. This is probably not correct for Sonata?